CalorieTracker/CalorieTracker/View Controllers/CalorieTrackerViewController.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  CalorieTrackerViewController.swift
+//  CalorieTracker
+//
+//  Created by Craig Swanson on 2/23/20.
+//  Copyright © 2020 Craig Swanson. All rights reserved.
+//
+
+import UIKit
+import CoreData
+import SwiftChart
+
+class CalorieTrackerViewController: UIViewController {
+
+    // MARK: - Properties
+    var calorieTrackerController = CalorieTrackerController()
+    
+    private lazy var fetchedResultsController: NSFetchedResultsController<CalorieTracker> = {
```

Comment:
This could be abstracted away. It is not the ViewController's responsibility to fetch information from core data context. Not a bit deal at all just something to consider for the future.